pkgs/by-name/ar/arrpc/package.nix

```diff
@@ -1,31 +1,39 @@
-{ lib
-, buildNpmPackage
-, fetchFromGitHub
-}: buildNpmPackage rec {
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+buildNpmPackage rec {
   pname = "arrpc";
-  version = "3.4.0";
+  version = "3.5.0";
 
   src = fetchFromGitHub {
     owner = "OpenAsar";
     repo = "arrpc";
-    # Release commits are not tagged
-    # release: 3.4.0
-    rev = "cca93db585dedf8acc1423f5e2db215de95c4c3b";
-    hash = "sha256-SeegrCgbjfVxG/9xfOcdfbVdDssZOhjBRnDipu6L7Wg=";
+    rev = version;
```

Comment:
```suggestion
    rev = "refs/tags/${version}";
```

Reply:
Why use the more tedious way? Normally I see `rev = "v${version}";`.

Reply:
`refs/tags/...` is the "correct" way to refer to tags on sources from GitHub.


The only place I've seen it mentioned in is [fetchgit reference](https://github.com/NixOS/nixpkgs/blob/master/doc/build-helpers/fetchers.chapter.md#fetchgit-fetchgit) but you will find that `refs/tags/...` is the encouraged format in most new PRs.

Reply:
Hm, but in the same documentation, it tells me to just use the tag name in `fetchFromGitHub`. I thought maybe the documentation at those places are written by different people, but no, they are introduced in the same commit (d7b62cb6).